e2e-tests/demand-backup-sharded/conf/some-name-rs0.yml

```diff
@@ -77,9 +77,7 @@ spec:
         replication:
           localPingThresholdMs: 15
       expose:
-        exposeType: ClusterIP
-        loadBalancerSourceRanges:
```

Comment:
I am not sure about it. I think we test loadBalancerSourceRanges in this place.

Reply:
It seems it might be true, so we can either move loadBalancerSourceRanges test to `demand-backup-physical-sharded` or move this check to this other test.